flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/stream/table/AsyncMLPredictITCase.java

```diff
@@ -111,6 +111,12 @@ public AsyncMLPredictITCase(
         content2vector.put(
                 Row.of("Fabian"),
                 Collections.singletonList(Row.of((Object) new Float[] {3.0f, 4.0f, 5.0f})));
+        content2vector.put(
```

Comment:
nit: Do we need change this?

Reply:
Doesn't hurt to add this. The original NPE was because of missing keys